hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsConfigKeys.java

```diff
@@ -265,4 +265,8 @@ private HddsConfigKeys() {
       "hdds.datanode.ratis.server.request.timeout";
   public static final String
       HDDS_DATANODE_RATIS_SERVER_REQUEST_TIMEOUT_DEFAULT = "2m";
+  public static final String HDDS_CONTAINER_CHECKSUM_ENABLED =
+          "hdds.container.checksum.enabled";
```

Comment:
Can we add "verification" to the config key?
```suggestion
          "hdds.container.checksum.verification.enabled";
```